jekyll/_cci2/server/installation/phase-3-execution-environments.adoc

```diff
@@ -1000,27 +1000,6 @@ databaseEncryption:
 …
 ----
 
-. *Create firewall rules*
-+
-Run the following commands to create a firewall rule for VM service in GKE:
-+
-NOTE: If you have used auto-mode, you can find the Nomad clients CIDR based on the region by referring to the https://cloud.google.com/vpc/docs/vpc#ip-ranges[table here].
-+
-[source,shell]
-----
-gcloud compute firewall-rules create "circleci-vm-service-internal-nomad-fw" --network "<network>" --action allow --source-ranges "0.0.0.0/0" --rules "TCP:22,TCP:2376"
-----
-+
-[source,shell]
-----
-gcloud compute firewall-rules create "circleci-vm-service-internal-k8s-fw" --network "<network>" --action allow --source-ranges "<clusterIpv4Cidr>" --rules "TCP:22,TCP:2376"
-----
-+
-[source,shell]
-----
-gcloud compute firewall-rules create "circleci-vm-service-external-fw" --network "<network>" --action allow --rules "TCP:54782"
```

Comment:
I agree with the removal of the above two, but won't the removal of 54782 result in customers not being able to SSH-rerun into VM-based jobs?